tests/end2end/test_single_operator_with_cntk_backend.py

```diff
@@ -171,6 +185,7 @@ def test_conv_4d(self):
 
         self._test_one_to_one_operator_coreml_channels_last(model, x)
 
+    @unittest.skipIf(not has_tensorflow(), reason="tensorflow is not installed")
```

Comment:
If every test has been added this skip, there must be something we should do differently. It's very easy to shadow the acutal test error without a notice.

Reply:
Given onnx and tensorflow are all not available for python 3.7, we can't claim onnxmltools has supported python 3.7, I'll remove this version from description and test configuration.

Reply:
The fix for python 3.7 has been merged into the main branch (https://github.com/tensorflow/tensorflow/pull/21202), so every module works on Python 3.7, the binaries are not release on pypi. I suggest keeping the tests.